packages/sample-server/test/credentialManifest.test.ts

```diff
@@ -1,48 +0,0 @@
-"use strict";
```

Comment:
why are tests being deleted?

Reply:
This is not the purpose of the sdk, it's a sample server to which we need to add some simple ui in order to complete the sample for sdk consumer.